lib/memjs/utils.js

```diff
@@ -72,6 +88,21 @@ exports.parseMessage = function(dataBuf) {
   return {header: responseHeader, key: key, extras: extras, val: val};
 };
 
+exports.parseMessages = function(dataBuf) {
+  var messages = [];
+
+  do {
+    var message = exports.parseMessage(dataBuf);
```

Comment:
I think the pattern looks like this:

```javascript
var message = undefined
while (message = exports.parseMessage(databuf)) {
  messages.push(message);
  var messageLength = message.header.totalBodyLength + 24;
  dataBuf = dataBuf.slice(messageLength);
}
```

Reply:
eslint doesn't like using assignment side effects for truthinesss